wagtail_footnotes/urls.py

```diff
@@ -1,12 +1,16 @@
 from django.urls import path
 from django.views.generic import TemplateView
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (4, 0):
+    template_name = "wagtail_footnotes/admin/footnotes_modals.html"
```

Comment:
There is a typo here, it should be singular "footnotes_modal.html".

Reply:
PR to fix this available in #44

Reply:
@jsma Thanks for doing that 👍